tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -2376,10 +2375,10 @@ def pad(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttl.tensor.pad(
```

Comment:
This file contains ttl operations for sweep tests, so this should stay as ttl or if pad in no longer available in ttl then this is no longer needed and we should remove pad tests. Please let @npetrovic-tenstorrent know if we are removing ttl pad so he can remove tests that use this.